config/config_test.go

```diff
@@ -71,6 +71,9 @@ txn-total-size-limit=2000
 [tikv-client]
 commit-timeout="41s"
 max-batch-size=128
+[tiflash]
+label-key="zone"
```

Comment:
Why we need `zone` and `engine` to identify a flash store?

Reply:
These two are defined by the flash team. By the way, label key or value can be configured in both sides(tiflash or tidb).